### PR TITLE
Run2-hcx183 Try remove some comments from Static Analyzer reported in #24361

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: SimG4CMS/Calo/interface/HcalTestAnalysis.h [hunk 1]
```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
...
-  HcalTestNumberingScheme   *org;
+  std::unique_ptr<HcalNumberingFromDDD>    numberingFromDDD_;
+  const HcalDDDSimConstants*               hcons_;
+  HcalTestNumberingScheme*                 org_;
 
   // Hits for qie analysis
-  std::vector<CaloHit>      caloHitCache; 
-  std::vector<int>          group_, tower_;
-  int                       nGroup, nTower;
+  std::vector<CaloHit>                     caloHitCache;
```
Comment (ianna) [inline SimG4CMS/Calo/interface/HcalTestAnalysis.h]: @bsunanda - please, rename all private members to have trailing underscore for consistency.

File: SimG4CMS/Calo/interface/HcalTestAnalysis.h [hunk 2]
```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
...
+  const HcalDDDSimConstants*               hcons_;
+  HcalTestNumberingScheme*                 org_;
 
   // Hits for qie analysis
-  std::vector<CaloHit>      caloHitCache; 
-  std::vector<int>          group_, tower_;
-  int                       nGroup, nTower;
+  std::vector<CaloHit>                     caloHitCache; 
+  std::vector<int>                         group_, tower_;
+  int                                      nGroup, nTower;
```
Comment (ianna) [inline SimG4CMS/Calo/interface/HcalTestAnalysis.h]: @bsunanda - please, rename all private members to have trailing underscore for consistency.

File: SimG4CMS/Calo/interface/HcalTestAnalysis.h [hunk 3]
```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
...
 
   // some private members for ananlysis 
-  unsigned int              count;                  
-  double                    edepEB, edepEE, edepHB, edepHE, edepHO;
-  double                    edepl[20];
-  double                    mudist[20];   // Distance of muon from central part
+  unsigned int                             count;                  
+  double                                   edepEB, edepEE, edepHB, edepHE;
+  double                                   edepHO, edepl[20];
+  double                                   mudist[20];   // Distance of muon from central part
```
Comment (ianna) [inline SimG4CMS/Calo/interface/HcalTestAnalysis.h]: @bsunanda - please, rename all private members to have trailing underscore for consistency.

File: SimG4CMS/Calo/interface/HcalTestAnalysis.h [hunk 4]
```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
...
+  int                                      centralTower_;
 
   // some private members for ananlysis 
-  unsigned int              count;                  
-  double                    edepEB, edepEE, edepHB, edepHE, edepHO;
-  double                    edepl[20];
-  double                    mudist[20];   // Distance of muon from central part
+  unsigned int                             count;                  
+  double                                   edepEB, edepEE, edepHB, edepHE;
+  double                                   edepHO, edepl[20];
```
Comment (ianna) [inline SimG4CMS/Calo/interface/HcalTestAnalysis.h]: @bsunanda - please, rename all private members to have trailing underscore for consistency.

File: SimG4CMS/Calo/interface/HcalTestAnalysis.h [hunk 5]
```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
...
+  double                                   eta0_, phi0_;
+  int                                      centralTower_;
 
   // some private members for ananlysis 
-  unsigned int              count;                  
-  double                    edepEB, edepEE, edepHB, edepHE, edepHO;
-  double                    edepl[20];
-  double                    mudist[20];   // Distance of muon from central part
+  unsigned int                             count;                  
+  double                                   edepEB, edepEE, edepHB, edepHE;
```
Comment (ianna) [inline SimG4CMS/Calo/interface/HcalTestAnalysis.h]: @bsunanda - please, rename all private members to have trailing underscore for consistency.

File: SimG4CMS/Calo/interface/HcalTestAnalysis.h [hunk 6]
```diff
@@ -62,36 +62,36 @@ class HcalTestAnalysis : public SimWatcher,
...
+  std::vector<std::string>                 names_;
+  double                                   eta0_, phi0_;
+  int                                      centralTower_;
 
   // some private members for ananlysis 
-  unsigned int              count;                  
-  double                    edepEB, edepEE, edepHB, edepHE, edepHO;
-  double                    edepl[20];
-  double                    mudist[20];   // Distance of muon from central part
+  unsigned int                             count;
```
Comment (ianna) [inline SimG4CMS/Calo/interface/HcalTestAnalysis.h]: @bsunanda - please, rename all private members to have trailing underscore for consistency.

**Review by ianna — CHANGES_REQUESTED** [review]

**Comment by andrius-k** [comment]
+1
Fine for DQM (no significant changes)

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Pull request merged**